adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -337,7 +337,9 @@ def MAC_address(self):        # pylint: disable=invalid-name
         if self._debug:
             print("MAC address")
         resp = self._send_command_get_response(_GET_MACADDR_CMD, [b'\xFF'])
-        return resp[0]
+        new_resp = bytearray(resp[0])
```

Comment:
check it! you can turn these three lines into one line of code by nesting the commands. please try it!

Reply:
@ladyada not sure what you are asking as the other functions follow the same format in line 337, 338.  Lines 337, 338, 339 are part of the original code so not sure where the efficiency is here.

Reply:
its not required, but sometimes people like to shorten it. how are you verifying the MAC address - with a router or...?

Reply:
@ladyada understood but the rest of the functions follow that convention so wanted to stay consistent for style as that would really freak me out seeing every other function in one coding style and this one in another format UBER OCD HERE!  My home router I only allow devices to which I specially add their MAC.  When I reversed the byte order I was successfully able to connect.   To confirm I also fired up my Kali box and did a netdiscover and clearly verified the reverse order sequence and the nice Expressif Inc. Vendor tag.

Reply:
oki!